projects/storefrontlib/src/styling/_variables.scss

```diff
@@ -1,67 +1,62 @@
-$white: #fff !default;
-$gray-100: #f8f9fa !default;
-$gray-200: #e9ecef !default;
-$gray-300: #dee2e6 !default;
-$gray-400: #ced4da !default;
-$gray-500: #adb5bd !default;
-$gray-600: #6c757d !default;
-$gray-700: #495057 !default;
-$gray-800: #343a40 !default;
-$gray-900: #212529 !default;
-$black: #000 !default;
+// VALUES USED AS EXAMPLES
 
-$blue: #007bff !default;
-$indigo: #6610f2 !default;
-$purple: #6f42c1 !default;
-$pink: #e83e8c !default;
-$red: #dc3545 !default;
-$orange: #fd7e14 !default;
-$yellow: #ffc107 !default;
-$green: #28a745 !default;
-$teal: #20c997 !default;
-$cyan: #17a2b8 !default;
+:host {
+  --cx-theme-primary: red;
+  --cx-theme-secondary: blue; //colors
+  --cx-theme-border-radius: 2em;
+  --cx-theme-transition-duration: 1s;
+  --cx-theme-animation-duration: 1s;
+  --cx-theme-gutter-width: 10px;
+  --cx-theme-font-weight-normal: 400; //bold, light
+  --cx-theme-spinner-size: 20px;
```

Comment:
this is not based on a css rule, can we make it more generic and try to mirror a css rule?